README.md

```diff
@@ -35,6 +36,17 @@ Docker container can be cleaned up using
 docker rm -f hintr
 ```
 
+### Input data
+
+Input data should be written to `/uploads` directory in the docker container, then when requesting validation pass the absolute path to the file in the request JSON e.g.
```

Comment:
I would rephrase this as "Input data should be written to the shared `upload_volume`, then when requesting validation pass the absolute path the file will be found at inside the docker container"